fog/ledger/server/src/bin/key_image_store.rs

```diff
@@ -32,7 +32,7 @@ fn main() {
     let _runtime_guard = _runtime.as_ref().map(|runtime| runtime.enter());
 
     let _tracer = mc_util_telemetry::setup_default_tracer_with_tags(
-        env!("CARGO_PKG_NAME"),
+        "fog_ledger_store",
```

Comment:
when debugging this it was a bit difficult when both of the services were called mc-fog-ledger

I'm up for suggestions here, but I think having separate names is better

<img width="1725" height="459" alt="Image" src="https://github.com/user-attachments/assets/17f51d30-f738-4e27-a971-df7e5a48341a" />